test/quantization/core/test_quantized_tensor.py

```diff
@@ -328,6 +329,82 @@ def test_qtensor_dtypes(self):
             rqr = qr.dequantize()
             self.assertTrue(np.allclose(r.numpy(), rqr.numpy(), atol=2 / scale))
 
+    @unittest.skipIf(not TEST_CUDA, "No gpu is available.")
+    def test_qtensor_to_device(self):
+        dtype_and_zero_types = [
+            (torch.quint8, torch.float),
+            (torch.qint8, torch.float),
+            #  (torch.qint32, torch.float) not supported for quantize_per_channel
+            (torch.quint8, torch.long),
+            (torch.qint8, torch.long),
+            (torch.qint32, torch.long),
+        ]
+        axis = 1
+        device = torch.device('cuda')
+        for dtype, zero_type in dtype_and_zero_types:
+            r = torch.rand(2, 2, dtype=torch.float) * 10
+            scales = torch.rand(2).abs()
+            zero_points = (torch.rand(2) * 10).round().to(zero_type)
+
+            # dqr = torch.quantize_per_channel(r, scales, zero_points, axis, dtype)
```

Comment:
nit: maybe we can split the quantize_per_tensor and quantize_per_channel tests to two to make it clearer?

Reply:
this is just testing to .to device functionality, its not actually testing the different quantization methods.